### PR TITLE
Update for support of riscvOVPsim

### PR DESCRIPTION
**Pull request by duncangraham-Imperas**
Tidied Makefile.include so it is consistent
Re-integrated the 64-bit device targets for riscvOVPsim
Add a postverify target to allow target specific post processing of data

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -89,8 +89,12 @@ simulate:
 verify:
 	riscv-test-env/verify.sh
 
-cover:
```
Comment (MarcKarasek) [inline Makefile:92]: why did u remove the cover: target?

**Review by MarcKarasek — COMMENTED** [review]
See inline comment

File: Makefile (hunk 1 above)
Comment (MarcKarasek) [reply Makefile:92]: If we remove this and add the postverify the documentation should also be changed/added to

Comment (neelgala) [reply Makefile:92]: The cover target was ovpsim specific to start with. I think moving it to the target makes sense.

Comment (Imperas) [reply Makefile:92]: Mark - I think Neel's comment covers it - so can this pull request please be merged. Thanks. Simon

**Review by MarcKarasek — APPROVED** [review]

**Pull request merged**